tests/Fixer/Comment/NoEmptyCommentFixerTest.php

```diff
@@ -66,10 +66,12 @@ public function provideFixCases()
             ),
             array(
                 '<?php
+                echo 2;
```

Comment:
Might make sense to omit them here, and PR this to 1.x. Also, it might be an idea to add a new test case rather than editing one so we have all cases properly covered, even if in the current implementation, it may be trivial to see that it works.